CUJExecutor/setCPUs

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+
+let "i = 0"
+
+for BIT in "$@"
+do
+	adb shell "echo $BIT > /sys/devices/system/cpu/cpu${i}/online 2> /dev/null"
```

Comment:
When and how will this be used?

Reply:
For now, this is just a helpful script for myself to test with setting CPUS to online/offline. (I lost it when I had to reset my container, so I re-wrote it and pushed it in case that happens again). Let me know if there's a better way to backup files that aren't necessarily "part of the project" just yet.